[ADD] - partner_routes: Add module

do-clearcorp opened the pull request:
Add: New module about add routes to the partner. Herency of the module res.partner, and add field partner_route_id (Many2one), herency of the module stock.picking and add field partner_route_id (Many2one), create method onchange.Change route when choosing the partner, generated spanish translation and security

Commit:
[ADD] - partner_routes: Add module

Add: New module about add routes to the partner. Herency of the module res.partner, and add field partner_route_id (Many2one), herency of the module stock.picking and add field partner_route_id (Many2one), create method onchange.Change route when choosing the partner, generated spanish translation and security

Commit:
[FIX] - partner_routes: Fix module

Fixed: The lines 14, 17 of stock_picking and space after import in __init__.py

gs-clearcorp commented on partner_routes/README.rst:
```diff
@@ -0,0 +1,26 @@
+Partner Routes
+=================
+
+This module adds Routes to the Partner.
```
Please add a detail description to the module

gs-clearcorp commented on partner_routes/README.rst:
```diff
@@ -0,0 +1,26 @@
+
+=================
+Partner Routes
+=================
```
Lines must match the string *Partner Routes* length (17 characters).

gs-clearcorp commented on partner_routes/__openerp__.py:
```diff
@@ -0,0 +1,30 @@
+
+{
+    'name': 'Partner Routes',
+    'summary': 'This is a module about of partner routes',
```
Please fix the module summary for example: *Add routes to customer*.

gs-clearcorp commented on partner_routes/partner_routes_menu.xml:
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © <YEAR(S)> ClearCorp
```
Add year to license

Commit:
[FIX] - partner_routes: Fixed module

Modified: added a detail description to the module, modified the lines must match the string Partner Routes length, fixed the module summary, added year to license
